### ISSUE TITLE
Time to put stuff on the web!

### ISSUE THREAD

**Issue by mjb232** [jlm323/DecameronProject#26]
@jlm323 @mmm202 Hey guys, so were almost done I just need to actually put all of our graphs and stuff on the web. So just make sure that your completed files and Ill get them to the web.

**Comment by jlm323** [comment]
My network graph is on here, uploaded as decamFlorenceImaginary.svg
I will write up a little explanation of the meanings of each color used and I'll send that to you ASAP.

Also, the map looks great! My only concern is that there seems to be "Naples" somewhere in India so the coordinates for that might be off. Also, we need to make sure we're adding the SSI to the Navigation/menu bar to each page so guests have a way to navigate from any page of the website. Otherwise, it looks good!

**Comment by jlm323** [comment]
Also, I just uploaded the JavaScript for the Index page so the trivia question should work once you get it on the web. Feel free to change the file name so it fits better.

**Comment by jlm323** [comment]
I added the explanation to the network graph. I just uploaded it here on github because I kept getting an error trying to update it in SSH /  upload it to the website.

Also did we pick a creative commons license? I suggest we go with this one:
Attribution-NonCommercial-ShareAlike 
CC BY-NC-SA

**Comment by mjb232** [comment]
I'll see if I can upload the page from my computer. Also I agree with your choice of CC

**Comment by mjb232** [comment]
@jlm323 I just added your description in, you are all good to go!

**Comment by jlm323** [comment]
@mjb232 Great, thanks!

**Comment by laurenmcguigan** [comment]
Hi guys, I don't know if any of you are still working, but I uploaded the day 1 output to Fetch. But how do I add a link in the index? can I do it from there. Or @mjb232 can you do it? It's uploaded to Fetch and it is called Day1output

**Comment by jlm323** [comment]
If you're trying to add a link to the navigation/menu bar, you can add it in decamnav.html

Also, I noticed we have a merge conflict

**Comment by ebeshero** [comment]
@laurenmcguigan I notice the site hasn't changed: trouble with editing and adding files to the site?

**Comment by laurenmcguigan** [comment]
@ebeshero  yes, I don't know which file is having a conflict, put we don't need to fix it tonight. It shouldn't be an issue in putting up my output. 

and 

@jlm323 How do I open it and make changes?

**Comment by jlm323** [comment]
It should be in your GitHub folder on your computer, under the Jessica-splashPage folder. Then just open that in Oxygen and add your changes, push the changes to GitHub and reupload it on Fetch

**Comment by ebeshero** [comment]
@jlm323 @laurenmcguigan I think I managed to resolved Lauren's merge conflict from my house! Lauren, your new pages are now in Jessica's repo, and in case there's a problem on your local directory, you want to just pull in the good version from Jessica:

```
git pull upstream master
```

Hope that works. 
Are you okay with posting to the website?

**Comment by ebeshero** [comment]
@laurenmcguigan I saw your boxModel.xhtml page in the website files and noticed it wasn't working--just showing the code when I look at the page. The problem is that you're missing the namespace declaration for xhtml. Your document node should look like this:

```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
```

Notice, I removed the xml line from the top, and that's because of what our Apache server expects at the PSC.
I also saved this simply as **.html** (not .xhtml). That's necessary for our Server Side Includes to work at the PSC. Please plan on saving all your web files as .html. 

Here's how your Very Cool boxModel page for the Decameron looks! 
http://decameron.newtfire.org/boxModel.html

You need to include the server side includes and the project CSS in addition to the CSS you wrote just for the page. OR you want to take the CSS for the page and write it as a separate page. I'm just going to add the `<link rel>` line to put the project CSS to work on it _in addition to_ the stuff you wrote inline. 

To reduce confusion, I'm going to rename Jessica's Splash Page directory to **"Website_Files"** in the main GitHub here, since we began building the website from in here! And I'll take the liberty of adding your very awesomely rendered Model Decameron page to the site. 
@mjb232 @jlm323 @mmm202

**Comment by ebeshero** [comment]
@laurenmcguigan I only added your box model page to the site... you'll need to add the rest. But have a look at how the Server Side Includes are set and make sure you understand the way the site is meshing together!
